kinesis/kinesis.go

```diff
@@ -317,7 +333,7 @@ func (outputPlugin *OutputPlugin) processAPIResponse(records *[]*kinesis.PutReco
 		*records = (*records)[:0]
 		*dataLength = 0
 	}
-	return fluentbit.FLB_OK, nil
+	return retCode, nil
```

Comment:
I think this has to do with my other question- help my understand how this changes the logic